### PR TITLE
Zachmu/update vitess

### PR DESCRIPTION
**Pull request by zachmu**

### CODE REVIEW

**Comment by zachmu** [comment]
Hold off on this -- I have some bugs to iron out

**Comment by Hydrocharged** [comment]
> 
> 
> Hold off on this -- I have some bugs to iron out

Darn it I was halfway through reviewing this lol

**Comment by zachmu** [comment]
While testing this, found that errors in tableEditor.close() permanently poison the session:

```
mysql> select * from a;
+------+----------------------------+
| x    | t                          |
+------+----------------------------+
|    1 | 2020-09-11 21:53:24.727333 |
|    2 | 2020-09-11 21:57:14.352303 |
|    3 | 2020-09-11 21:57:14.352303 |
|    4 | 2020-09-11 21:57:14.352303 |
|   10 | 2020-09-12 00:08:13.993816 |
+------+----------------------------+
5 rows in set (0.00 sec)

mysql> insert into a values (11, now());
ERROR 1105 (HY000): unknown error: failed to write table back to database
mysql> insert into a values (12, now());
ERROR 1105 (HY000): unknown error: failed to write table back to database
```

**Comment by zachmu** [comment]
Since the breaking bug I found actually exists on the earlier version of dolt, let's get this checked in and you can fix it off of master instead of this branch.

File: go/libraries/doltcore/doltdb/table_editor.go [hunk 1]
```diff
@@ -440,6 +443,37 @@ func (te *TableEditor) flushEditAccumulator(ctx context.Context, teaInterface in
 	return nil
 }
 
+// formatKey returns a comma-separated string represnetation of the key given.
```
Comment (Hydrocharged) [inline go/libraries/doltcore/doltdb/table_editor.go:446]:
```suggestion
// formatKey returns a comma-separated string representation of the key given.
```

File: go/libraries/doltcore/sqle/sqlbatch_test.go [hunk 2]
```diff
@@ -200,9 +200,9 @@ func TestSqlBatchInsertErrors(t *testing.T) {
 	assert.NoError(t, drainIter(rowIter))
 
 	// This generates an error at insert time because of the bad type for the uuid column
-	_, _, err = engine.Query(sqlCtx, `insert into people values
+	_, rowIter, err = engine.Query(sqlCtx, `insert into people values
```
Comment (Hydrocharged) [inline go/libraries/doltcore/sqle/sqlbatch_test.go:203]: What about the error here?

**Review by Hydrocharged — APPROVED** [review]

**Pull request merged**